server/MapGo_server/urls.py

```diff
@@ -13,9 +13,25 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
+from django.conf.urls import include
 from django.contrib import admin
 from django.urls import path
 
+from rest_framework import routers
+
+from checkin.views import CheckinViewSet
+from recommend.views import STCViewSet
+
+router = routers.DefaultRouter()
+router.register('Mapgo/checkin', CheckinViewSet, basename="checkin")
+
+router2 = routers.DefaultRouter()
+router2.register('Mapgo/recommend', STCViewSet, basename='recommend')
+
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('', include(router.urls)),
+    path('Mapgo/checkin/<str:userid>/',
+         CheckinViewSet.as_view({'delete': 'delete'})),
+    path('', include(router2.urls)),
```

Comment:
@lijm1358 `router1`, `router2` namings are seems little bit ambiguous to me.
How about use dictionary?
```suggestion
routers = {
    "checkin" : routers.DefaultRouter()
    "recommend" : routers.DefaultRouter()
}

routers["checkin"].register('Mapgo/checkin', CheckinViewSet, basename="checkin"),
routers["recommend"].register('Mapgo/recommend', STCViewSet, basename='recommend')

urlpatterns = [
    path('admin/', admin.site.urls),
    path('', include(routers["checkin"].urls)),
    path('Mapgo/checkin/<str:userid>/',
         CheckinViewSet.as_view({'delete': 'delete'})),
    path('', include(routers["recommend"].urls)),
```